src/audio/AudioManager.cpp

```diff
@@ -81,6 +84,7 @@ AudioManager::AudioManager(){
 *   Description:
 *       Desctructor of AudioManager. Frees memory and quits SLD mixer
 */
+#ifndef SERVER
```

Comment:
oh no, this is not right at all.
they are supposed to go only _inside_ the function calls to prevent it from running when compiled as server.

please do a bit of research into how to use these so you know